projects/fsastorefront-e2e-cypress/cypress.json

```diff
@@ -1,12 +1,13 @@
 {
   "defaultCommandTimeout": 10000,
-  "baseUrl": "https://fsastorefrontapp.c73z0k1qfh-financial1-s6-public.model-t.cc.commerce.ondemand.com",
+  "baseUrl": "https://fsastorefrontapp.c73z0k1qfh-financial1-s5-public.model-t.cc.commerce.ondemand.com",
```

Comment:
Just revert this to local instance. When you are writing tests, just change it on your local instance, shouldn't be commited like this

Reply:
we have two cypress json files, one for test systems, and separate one for local cypress.local.json

Reply:
Fix then both to local :D